src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -324,9 +318,31 @@ public WebView(WebView webView) : this(Interop.WebView.NewWebView(WebView.getCPt
         [EditorBrowsable(EditorBrowsableState.Never)]
         public WebSettings Settings { get; }
 
+        /// <summary>
+        /// The uri to load.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Please add `privilege` and remove `[EditorBrowsable(EditorBrowsableState.Never)]`.

Reply:
Done.